programs/svm-spoke/src/state/state.rs

```diff
@@ -12,7 +12,7 @@ pub struct State {
     pub current_time: u32,              // Only used in testable mode, else set to 0 on mainnet.
     pub remote_domain: u32,             // CCTP domain for Mainnet Ethereum.
     pub cross_domain_admin: Pubkey,     // HubPool on Mainnet Ethereum.
-    pub root_bundle_id: u32,            // TODO rename to next_root_bundle_id
+    pub root_bundle_id: u32,            // Tracks the current root bundle id.
```

Comment:
but it is not the current root bundle id. It is next one, thus we had comment to rename this prop

Reply:
I chose to just update the comment accordingly.